hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataReader.java

```diff
@@ -323,8 +323,11 @@ public List<OmKeyInfo> listKeys(String volumeName, String bucketName,
             bucket.realVolume(), bucket.realBucket(), keyPrefix);
       }
       metrics.incNumKeyLists();
-      return keyManager.listKeys(bucket.realVolume(), bucket.realBucket(),
-          startKey, keyPrefix, maxKeys);
+      List<OmKeyInfo> omKeyInfoList =
```

Comment:
Should this metric be captured outside to take every cost, including bucket resolving and ACL check into account? Seems like we need a simple breakdown of the metric.

Reply:
I agree - I have made the changes, thank you @duongkame!